components/03-forms/01-groups/group-files.njk

```diff
@@ -0,0 +1,37 @@
+<div {{ className('form-group', 'dso-files', [required, 'dso-required']) }}>
+  <div class="dso-description dso-rich-content">
+    {{ description }}
+  </div>
+  <ul class="dso-filelist">
+    {% for file in files %}
+      {% render '@file-item', {filename: file.filename, pos: loop.index0, confidential: file.confidential} %}
+    {% endfor %}
+  </ul>
+  {% for file in files %}
+    {% if file.confidential %}
```

Comment:
In de Handlebars template was een notatie gemaakt zodat het op 1 regel pastte EN er alleen werd geïtereerd.

Kan je het proberen met een nieuw filter? Ik vind een `map` filter wel cool

Reply:
Kan je een voorzet geven? Ik heb hier al aardig wat tijd in zitten en was al blij dat dit werkt.